docs/eventing/experimental-features/sender-identity.md

```diff
@@ -0,0 +1,233 @@
+# Sender Identity for Knative Eventing
+
+**Flag name**: `authentication-oidc`
+
+**Stage**: Alpha, disabled by default
+
+**Tracking issue**: [#6806](https://github.com/knative/eventing/issues/6806)
+
+## Overview
+
+Currently, event delivery within the cluster is unauthenticated, and an addressable event consumer cannot determine the identity of any sender.
+
+Knative Eventing Addressables expose their OIDC Audience in their status as part of their address (e.g. `.status.address.audience`) and require requests containing an OIDC access token issued for this audience.
+
+Knative Eventing Souces request an OIDC access token for the targets audience and add them to the request. A per-source dedicated Service Account is used as the identity for the request.
+
+## Prerequisites
+
+- [Eventing installation](./../../install)
+
+## Sender Identity Configuration
+
+The possible values for `authentication-oidc` are:
+
+- `disabled`
+    - No change in behaviour
+- `enabled`
+    - Addressables announce their audience in their status
+    - Sources add an Authorization Header to their request containing an access token for their target
+
+For example, to enable sender identity, the `config-features` ConfigMap will look like
+the following:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-features
+  namespace: knative-eventing
+data:
+  authentication-oidc: "enabled"
```

Comment:
do we want to mention that TLS should also be enabled?

Reply:
Good point. I added a note about TLS too